Automattic/wp-calypso
# Domains: Update Domain Transfer checkout page

#### Changes proposed in this Pull Request
This PR implements the changes on the domain transfer checkout page layout. 
Here's a comparison of how they looked before vs. how they're looking now: 

#### Desktop

##### Before
<img width="900" alt="before - desktop" src="https://user-images.githubusercontent.com/18705930/129769526-cb639bc4-1a05-42b3-8c3b-5265156914fa.png">

##### After
<img width="900" src="https://user-images.githubusercontent.com/18705930/131395233-e76b4516-a179-4fe2-a6bb-0fdae4b82585.png"/>

#### Mobile
##### Before
<img width="400" alt="before - mobile" src="https://user-images.githubusercontent.com/18705930/129769644-05a8a46d-880e-47dd-b2e4-135da58cf5d2.png">

##### After
<img width="400" src="https://user-images.githubusercontent.com/18705930/131515497-093db194-6e01-497a-bc0d-cc63e0df7bdb.png"/>


#### Testing instructions
 - Add a domain transfer to the cart;
 - Checkout and confirm that the new "thank you" page looks like this. 

_(If you already have a domain transferred and you have the receipt ID, you can bypass the checkout by going to `/checkout/thank-you/:your-site/:receipt-id` 😉)_
